recipes-kernel/perf/perf.bbappend

```diff
@@ -2,4 +2,4 @@
 # NILRT ARM images use linux kernel 4.14 - which does not support perf
 # scripting using python3. Disable `scripting` PACKAGECONFIG in that case, to
 # keep python3 out of the perf config.
-PACKAGECONFIG:remove:xilinx-zynq += "scripting"
+PACKAGECONFIG:remove:xilinx-zynq = "scripting"
```

Comment:
I *think* this should be fine; but check for safety.

Reply:
From `bitbake -e perf`, `PACKAGECONFIG` contains `scripting` as expected since this change only affects arm.